mock/src/mock/chain.rs

```diff
@@ -660,17 +660,15 @@ pub fn mock_chain_data(consumed_notes: Vec<Note>) -> (ChainMmr, Vec<InputNote>)
 // ================================================================================================
 
 /// Converts the MMR into partial MMR by copying all leaves from MMR to partial MMR.
-fn mmr_to_chain_mmr(mmr: &Mmr) -> ChainMmr {
+fn mmr_to_chain_mmr(mmr: &Mmr, blocks: &[BlockHeader]) -> ChainMmr {
```

Comment:
Why not have the function take as input `blocks: Vec<BlockHeader>` so we would not have to use `blocks.to_vec()`